### DISCUSSION TITLE
Using multiple image loaders

### DISCUSSION THREAD

**Discussion by samrobbins85** [vercel/next.js#23820] [Help]
From the [documentation of the loader](https://nextjs.org/docs/basic-features/image-optimization#loader), it looks like you can just specify one loader for all the images. From what I remember the loader prop on the image component used to allow you to specify the premade loaders, but [now appears](https://nextjs.org/docs/api-reference/next/image#loader) to just allow you to write your own function.

Is there a way I can specify that I want to use the imgix loader for one image component or do I have to write it as a function?

**Comment by akellbl4** [comment]
As I can see by the source code of `next/image` you can specify the only function as the prop. I think you can get the code of the loaders that you need and use them for the particular situation.
